### ISSUE TITLE
Fix "Get Token" Error

### ISSUE THREAD

**Issue by tylerharter** [pivot-libre/pivot#97]
When the button is clicked but the Laravel mail driver is not configured, I can see that the API call returns a string: "mail driver not configured".  But the message displayed says "An issue occurred when generating the token. Check that you have supplied a valid email address."

Should we just print whatever error the backend returns?  I imagine this will be more useful when the backend returns something like "an account already exists for that email" (I haven't implemented that yet).

**Comment by neckberg** [comment]
Yeah, that other text was just a placeholder, since the current response from the server when you supply a bad email is not user friendly. For now, I've updated it to show the response from the server.
